cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -46,7 +48,7 @@ def setUp(self):
         self.client = AjaxEnabledTestClient()
         self.client.login(username=self.user.username, password='test')
 
-    def _create_course_with_access_groups(self, course_location, user=None):
+    def _create_course_with_access_groups(self, course_location, user=None, store=ModuleStoreEnum.Type.mongo):
```

Comment:
The default store should be split (I know that's changing functionality, but if tests don't fail, this will save us a world of hurt down the line)

And if tests do fail, then we should file a ticket to change the default store here to split.